packages/css-data/src/parse-css.ts

```diff
@@ -113,6 +118,63 @@ const cssTreeTryParse = (input: string) => {
   }
 };
 
+const convertBackgroundProps = (styles: EmbedTemplateStyleDecl[]) => {
```

Comment:
@TrySound wasn't this supposed to be done by the short>long converter

Reply:
All good, our parser just don't understand lists